lib/config.js

```diff
@@ -29,6 +29,12 @@ cfg.supportedSuites = [
   'eddsa-2022',
 ];
 
+// parameters for verifier specific did drivers
+cfg.methods = {
```

Comment:
Add README entries for this config that link back to `did-method-web`.

Reply:
Added README entry here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/44/commits/0fcb624e4e60fb56549154e255968b237fd4ae8b